frontend/src/app/components/Restart.js

```diff
@@ -13,30 +13,21 @@ export default class Restart extends React.Component {
     });
   }
 
-  renderSubtitle() {
-    const { experimentTitle } = this.props;
-    if (experimentTitle) {
-      return (
-        <h2 data-l10n-id="restartRequiredFromExperiment">
-           data-l10n-args={JSON.stringify({ experimentTitle })}
```

Comment:
We do [some work](https://github.com/mozilla/testpilot/search?utf8=%E2%9C%93&q=experimentTitle) to make `experimentTitle` available here. If we won't use it anymore, let's pull all that out.